docs/components/icon.md

```diff
@@ -76,6 +76,83 @@ You can read more about the viewBox attribute
 <KIcon icon="portal" viewBox="0 0 10 10" />
 ```
 
+
+## Slots
+- `svgElements` - Used to add svg customization elements
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏`svgElements` is an overlay on the icon?